www/site/content/docs/chef-workstation/chef-run-users-guide.md

```diff
@@ -36,7 +36,7 @@ One choice for specifying your `chef-run` command is:
 chef-run host1 directory /tmp/foo
 ```
 
-To use password authentication instead of an identity file, specify it as part of the connection information or using the command line flag:
+To use password authentication instead of an identity file, specify the identity file location as part of the connection information or using the command line flag:
```

Comment:
Should be: `specify the password as part of the connection information or using the command line flag`

(Which highlights why the change was needed in the first place :D)